workflows/src/main/resources/migration/allowed-to-see-states-schema.changelog-init.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="R3.Corda" id="create_allowed_to_see_states_table_id">
+        <createTable tableName="account_to_state_refs">
+            <column name="id" type="BIGINT">
+                <constraints nullable="false"/>
```

Comment:
All these columns have a non-null constraint on them in the liquibase migration, but I think the corresponding JPA entity allows for these to be null. Probably the entity needs adjusting?